src/main/java/FindCommand.java

```diff
@@ -0,0 +1,33 @@
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+    String keyword;
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        ArrayList<Task> reqTasks = tasks.find(this.keyword);

```

Comment:
Might be a good idea to abstract this in a method in UI, like showMessage or something